config/toolchain.in

```diff
@@ -117,11 +117,12 @@ config TOOLCHAIN_BUGURL
       Specify the URL that users should visit if they wish to report a bug.
 
 comment "Tuple completion and aliasing"
+    depends on !TOOLCHAIN_STRIP_VENDOR
 
 config TARGET_VENDOR
     string
     prompt "Tuple's vendor string"
-    depends on !LIBC_AVR_LIBC
+    depends on !LIBC_AVR_LIBC && !TOOLCHAIN_STRIP_VENDOR
```

Comment:
Rather, make `avr-libc.in` do a `select TOOLCHAING_STRIP_VENDOR` so that there's a single option controlling this behavior.